recipe/meta.yaml

```diff
@@ -1,18 +1,19 @@
 {% set name = "nitime" %}
-{% set version = "0.9" %}
+{% set version = "0.10.1" %}
+{% set sha256 = "367a15ad2b7a3134dc36ea757beff5bf92681c261cc9cb901f8ac72f35c9b7e6" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/nipy/nitime/archive/refs/tags/rel/{{ version }}.tar.gz
-  sha256: badf4b5b6940f339c39a2fa873b1dcb44e41f4294f72a0e1d163881a1f973a69
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Increasingly the convention to use PyPI sdists as the source for wheels, conda packages and Linux distro packages.

Reply:
Not really. We use the best source available. Sometimes the PyPI sdist is missing files, license, etc. If that is not the case here we can use the PyPI sdist, no problem.

Reply:
I was guessing the reason the autotick bot didn't attempt to upgrade was because it was pointing to a GitHub archive instead of PyPI. Do you know the rules?

In any case, the sdist does contain the license, and the package is PEP-517 compliant so wheels are now generated from the sdist.

Reply:
> I was guessing the reason the autotick bot didn't attempt to upgrade was because it was pointing to a GitHub archive instead of PyPI. Do you know the rules?

If there are no new tags on GH that may be the cause but this tag is there, see https://github.com/nipy/nitime/releases/tag/0.10.1. However, I checked the conda-forge status page and it seems that the bot is not smart enough to move to a tag URL when a release URL is expected. See

![Screenshot from 2023-05-24 14-24-17](https://github.com/conda-forge/nitime-feedstock/assets/950575/24cb8692-1a35-4da7-9807-3d16b756bf1d)


TL;DR moving the the PyPI one here is fine but we, conda-forge, should make this more flexible and take both tag and releases URLs for GH udpates.